.kokoro-windows/New-BuildEnv.ps1

```diff
@@ -22,36 +22,41 @@ get-command choco -ErrorAction Stop
 choco list -li | Tee-Object -Variable chocoList
 $chocoPackages = ($chocoList) -join ' '
 
-if (-not $chocoPackages.Contains('Microsoft .NET Core SDK - 2.1.4')) {
-    choco install -y --sxs dotnetcore-sdk --version 2.2.203
+if (-not $chocoPackages.Contains('Microsoft .NET Core SDK - 2.2.')) {

```

Comment:
Not sure we should be using 2.2 at all - that went out of support even longer ago than 2.1.

Can we get rid of it entirely?

Reply:
There are like 10 sample applications that are using 2.2. Those are all web application so moving them away from 2.2 takes time.
We have:

- #1548 
- #1004